src/exports.jl

```diff
@@ -1065,7 +1068,8 @@ export negbias
 export negdeglex
 export negdegrevlex
 export neglex
-export negrevlex
+export neginvlex
+export negrevlex # to be removed
```

Comment:
```suggestion
```
neginvlex needs to be moved up one line, and `negrevlex` should be exported by `@deprecate` already